cellmlmanip/parser.py

```diff
@@ -310,41 +312,49 @@ def _add_variables(self, component_element):
 
         return variable_lookup_symbol
 
-    def _add_maths(self, component_element, variable_to_symbol):
+    def _add_maths(self, maths_to_add, connected_variable_mapping):
         """
+        Add maths for all elements sorted in maths_to_add.
+
         <model> <component> <math> </component> </model>
 
-        :param component_element: an etree.Element
-        :param variable_to_symbol: a ``Dict[str, sympy.Dummy]``
+        :param maths_to_add: a list of (component_element, variable_to_symbol) tuples
```

Comment:
Great docs!

Reply:
@MichaelClerx let me know if you have any constructive feedback thanks!

Reply:
It was genuine, I was pleased to see it!